dev/com.ibm.ws.container.service/src/com/ibm/wsspi/application/handler/ApplicationInformation.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2012, 2025 IBM Corporation and others.
```

Comment:
Not sure why a new class would have a starting date of 2012.  What is the value of having this added to the SPI.  How does a user get an instance of this interface? Can it be moved to a private package instead?

Reply:
If you plan to keep this class, the package-info for this package should be updated to have the second digit updated.